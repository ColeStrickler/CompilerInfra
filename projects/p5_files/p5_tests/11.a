b:bool;

g : () -> bool {
    b = 6 == 6;
    b++;
    return b;
}
