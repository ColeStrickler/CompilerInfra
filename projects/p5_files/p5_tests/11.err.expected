FATAL [5,5]-[5,8]: Arithmetic operator applied to invalid operand
Type Analysis Failed
